pyqir/tests/test_string_attributes.py

```diff
@@ -77,3 +81,84 @@ def test_attribute_alphabetical_sorting() -> None:
     # Tests that attributes are sorted alphabetically by key,
     # irrespective of their value
     assert 'attributes #0 = { "1" "A"="123" "a"="a" "b"="A" "c" }' in ir
+
+
+def test_function_attributes_can_be_iterated_in_alphabetical_order() -> None:
+    mod = pyqir.Module(pyqir.Context(), "test")
+    void = pyqir.Type.void(mod.context)
+    function = Function(FunctionType(void, []), Linkage.EXTERNAL, "test_function", mod)
+    # add them out of order, they will be sorted automatically
+    add_string_attribute(function, "required_num_results", "1")
+    add_string_attribute(function, "entry_point", "")
```

Comment:
Could this be `add_string_attribute(function, "entry_point")` since value is optional? Or in this specific test case you want the value to be an empty string?